metals/src/main/scala/scala/meta/internal/metals/codeactions/ImportMissingSymbol.scala

```diff
@@ -43,14 +44,42 @@ class ImportMissingSymbol(compilers: Compilers) extends CodeAction {
         }
     }
 
+    def addImportAllMissingSymbol(
+        codeActions: Buffer[l.CodeAction]
+    ): Seq[l.CodeAction] = {
+      val uniqueCodeActions = codeActions
+        .groupBy(_.getDiagnostics())
+        .values
+        .filter(_.length <= 1)
```

Comment:
What would be a scenario where we'd want `< 1`? Aren't we always targeting when there is `1` exactly?

Reply:
yep that's true